rules/python-click.mk

```diff
@@ -7,11 +7,10 @@
 # TODO: If we upgrade to a distro which provides a version >= 6.7 we will no
 # longer need to build this.
 
-PYTHON_CLICK_VERSION = 6.7-4
+PYTHON_CLICK_VERSION = 7.0-1
```

Comment:
Per my comment above, if the distro provides python-cli .deb package >= 6.7, it will contain the bug fixes we need, so we no longer need to build the package from source. Buster provides version 7.0-1.

Reply:
to make this pr simple, this will be address after buster merge.